src/hotspot/share/gc/shared/workgroup.hpp

```diff
@@ -26,6 +26,7 @@
 #define SHARE_GC_SHARED_WORKGROUP_HPP
 
 #include "memory/allocation.hpp"
+#include "metaprogramming/logical.hpp"
```

Comment:
Should also add metaprogramming/enableIf.hpp, to avoid implicit include dependency.

Reply:
Should I include both `enableIf.hpp` and `logical.hpp`, or only `enableIf.hpp` since it includes `logical.hpp` already?

Reply:
We don't have any tooling support, but there has been a trend away from relying on implicit includes, because they lead to breakages far away from refactorings that change includes.

Reply:
Got it; including both then.